libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/Spec/Conway.hs

```diff
@@ -17,7 +17,7 @@ spec = describe "Conway conformance tests" $ do
   xprop "UTXO" $ conformsToImpl @"UTXO" @ConwayFn @Conway
   prop "GOV" $ conformsToImpl @"GOV" @ConwayFn @Conway
   prop "CERT" $ conformsToImpl @"CERT" @ConwayFn @Conway
-  xprop "RATIFY" $ conformsToImpl @"RATIFY" @ConwayFn @Conway
+  prop "RATIFY" $ conformsToImpl @"RATIFY" @ConwayFn @Conway
```

Comment:
Leftover from the other PR,  if that one gets merged first I guess it will disappear on its own

Reply:
Nevermind, sorry, was reviewing by commit.